api/interfaces/BuildInterfaces.ts

```diff
@@ -11,8 +11,8 @@
 "use strict";
 
 import DistributedTaskCommonInterfaces = require("../interfaces/DistributedTaskCommonInterfaces");
-import TFS_SourceControl_Contracts = require("./TfvcInterfaces");
-import TFS_TestManagement_Contracts = require("./TestInterfaces");
+import TFS_SourceControl_Contracts = require("../interfaces/TfvcInterfaces");

```

Comment:
Did we move these on purpose or is it because of regen changes?

Reply:
Just the regen, it has no practical impact though

Reply:
Do we not use those files here? How can we change the path but not move the file and it still works?

Reply:
The path is basically the same, we just change `./` to `../interfaces/` which is fine since we're already in the `interfaces` folder and they both just resolve to `./`